src/rgw/rgw_sync_module_es.cc

```diff
@@ -248,14 +248,15 @@ struct es_index_config {
 };
 
 static bool is_sys_attr(const std::string& attr_name){
-  static constexpr std::initializer_list rgw_sys_attrs = {RGW_ATTR_PG_VER,
-                                                          RGW_ATTR_SOURCE_ZONE,
-                                                          RGW_ATTR_ID_TAG,
-                                                          RGW_ATTR_TEMPURL_KEY1,
-                                                          RGW_ATTR_TEMPURL_KEY2,
-                                                          RGW_ATTR_UNIX1,
-                                                          RGW_ATTR_UNIX_KEY1
-  };
+  static constexpr std::initializer_list<char[64]> rgw_sys_attrs = 
```

Comment:
```suggestion
  static constexpr std::initializer_list<const char*> rgw_sys_attrs =
  {                                                       RGW_ATTR_PG_VER,
```

does this work. also, i'd suggest minimize the code churn to fix the FTBFS with Clang.

Reply:
@tchaikov 
That seems to please Clang
Will update PR